test/compilable/fix19018.d

```diff
@@ -0,0 +1,22 @@
+/* REQUIRED_ARGS: -dw
```

Comment:
FWIW we typically use `-de` here and put the test in fail_compilation.
The advantage is that once the deprecation is over, the test doesn't need to be moved, but there's no strict rule on this.